client/cli/go/cmds/utils.go

```diff
@@ -111,3 +111,25 @@ func newHeketiClient() (*client.Client, error) {
 			VerifyCerts:        options.TLSCerts,
 		})
 }
+
+const (
+	KB uint64 = 1 << (10 * (iota + 1))
+	MB
```

Comment:
I am not sure this works as intended. Should not all values be set here?

Reply:
This notation also works as expected. Example here https://play.golang.org/p/Jrom7MxELGj

Reply:
No, this is fine imho. My reaction has been added as a [comment further below](#issuecomment-395374362).

Reply:
Used https://github.com/golang/go/wiki/Iota#examples to beautify declarations. Actually `(iota+1)` not needed.